asm/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since asm is a new module, you'll need to add it to the root-level pom.xml, so that it gets built during the main build

Reply:
Done